.github/ISSUE_TEMPLATE/formatting.md

```diff
@@ -0,0 +1,44 @@
+---
+name: ✨ Formatting
+about: Issues for ugly or incorrect code
+---
+
+<!--
+
+BEFORE SUBMITTING AN ISSUE:
+
+1.  Search for your issue on GitHub: https://github.com/prettier/plugin-php/issues
+    A large number of opened issues are duplicates of existing issues.
+    If someone has already opened an issue for what you are experiencing,
+    you do not need to open a new issue — please add a 👍 reaction to the
+    existing issue instead.
+
+2.  We get a lot of requests for adding options, but Prettier is
+    built on the principle of being opinionated about code formatting.
+    This means we have a very high bar for adding new options.
+    Find out more: https://prettier.io/docs/en/option-philosophy.html
```

Comment:
Might make sense to move this section to the `README` instead?

Reply:
Yep, let improve README in other PR, i think we should add many thing to readme

Reply:
@czosel yep, :+1:

Reply:
@evilebottnawi Wanna create one? :wink:

Reply:
@czosel i am not good in English, better when you do it :smile:

Reply:
:laughing: I meant just an issue to collect some ideas about what should be added to the README. Once I know what to write, I can make a PR :wink:

Reply:
:+1: